src/interface.ts

```diff
@@ -44,19 +41,29 @@ export namespace UI {
             openLabel: "Select an APK file",
         });
         if (result && result.length === 1) {
-            const args = await showArgsQuickPick(
+            const quickPickItems = await showArgsQuickPick(
                 quickPickUtil.getQuickPickItems("decodeQuickPickItems"),
                 "Additional features & Apktool/Jadx arguments"
             );
-            if (args) {
-                const decompileJavaIndex = args.indexOf("decompile_java");
-                const quarkAnalysisIndex = args.indexOf("quark_analysis");
-                const jadxOptionsIndex = args.indexOf("--deobf");
+
+            if (quickPickItems) {
+                const args = quickPickItems.map<string>((item) => item.label);
+                const argsIndex = quickPickItems.map<string | undefined>(
```

Comment:
Index might reflect the intent but not super clear right away.
```suggestion
                const argDescriptions = quickPickItems.map<string | undefined>(
```